config/features.yml

```diff
@@ -202,9 +202,6 @@ features:
   decision_review_sc_email:
     actor_type: user
     description: Send email notification for successful SC submission
-  decision_review_hlr_status_update_throttling:
-    actor_type: user
-    description: Throttles requests to CMP when updating HLR statuses
```

Comment:
These lines were removed because the `decision_review_hlr_status_update_throttling` feature was found nowhere else in the code base. I expect it was removed at some point but the engineer just forgot to remove it from this file.

Reply:
I think that was me that left it in :( Sorry about that!

Reply:
Oh, no worries!